common/baselines/librispeech/ls100/gmm/baseline_args.py

```diff
@@ -0,0 +1,428 @@
+"""
+Definition of all changeable pipeline parameters of the GMM-RASR-Pipeline
+
+The parameters defined here are based on the past experience with training LibriSpeech-100h and 960h models,
+but no excessive tuning has been done.
+"""
+from i6_core.features.filterbank import filter_width_from_channels
+from i6_core import cart
+
+from i6_experiments.common.setups.rasr import gmm_system
+
+from i6_experiments.common.setups.rasr.util import *
```

Comment:
sorry, I know you explicitly asked to not comment on this, but can we be explicit on which things are imported from where?